orchagent/srv6orch.cpp

```diff
@@ -600,6 +784,328 @@ bool Srv6Orch::deleteMysidEntry(const string my_sid_string)
     return true;
 }
 
+uint32_t Srv6Orch::getAggId(const NextHopGroupKey &nhg)
+{
+    SWSS_LOG_ENTER();
+    static uint32_t g_agg_id = 1;
```

Comment:
instead of having static g_agg_id inside a member function, can we move this to srv6Orch object?

Reply:
Fixed, changed to member "m_srv6_agg_id".

Reply:
Since there is no "index allocator" library in orch and we don't want to introduce dependency by new library, we write this simple function to get the id. And in our system level test, there is no performance issue. Is it ok to keep this function?